setup.py

```diff
@@ -1,31 +1,14 @@
-#!/usr/bin/env python3
-import logging
 import re
-import sys
 from pathlib import Path
 
 from setuptools import find_packages
 from setuptools_scm import get_version
 from skbuild import setup
 
 __version__ = get_version(root=".", relative_to=__file__)
-
-logging.basicConfig(level=logging.INFO)
-log = logging.getLogger("cuvec.setup")
-
 build_ver = ".".join(__version__.split('.')[:3]).split(".dev")[0]
-cmake_args = [f"-DCUVEC_BUILD_VERSION={build_ver}"]
-try:
-    from miutil import cuinfo
-    nvcc_arch_raw = map(cuinfo.compute_capability, range(cuinfo.num_devices()))
-    nvcc_arches = {"%d%d" % i for i in nvcc_arch_raw if i >= (3, 5)}
```

Comment:
this error-prone hack (hardcoding min compute capability `3.5` rather than using proper tables[^1][^2][^3]) is required due to https://gitlab.kitware.com/cmake/cmake/-/issues/24921

[^1]: https://stackoverflow.com/questions/28932864/which-compute-capability-is-supported-by-which-cuda-versions
[^2]: https://stackoverflow.com/questions/30820513/what-is-the-correct-version-of-cuda-for-my-nvidia-driver
[^3]: https://docs.nvidia.com/cuda/cuda-compiler-driver-nvcc/index.html#gpu-generations